### ISSUE TITLE
`--private-ip-address` does nothing during Application Gateway w/ WAF_v2 creation

### ISSUE THREAD

**Issue by alexpilon666** [Azure/azure-cli#14494]
When using `az network application-gateway create` with the `--private-ip-address` argument, Azure CLI throws no errors but also does not create/associate the Private IP address.

Without this argument doing anything, I have to then use a second command, `az network application-gateway frontend-ip create` to create the private frontend IP configuration, which adds extra time and steps.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d85e1042-f9fe-a899-7a64-99796558a7aa
* Version Independent ID: 2573afce-d2fa-6f5f-3647-ea8696b8d0e2
* Content: [az network application-gateway](https://docs.microsoft.com/en-us/cli/azure/network/application-gateway?view=azure-cli-latest#az-network-application-gateway-create)
* Content Source: [src/azure-cli/azure/cli/command_modules/network/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/network/_help.py)
* Service: **virtual-network**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
add to S174

**Comment by gvdm90** [comment]
Hi,
I found this same bug running this command
```
az network application-gateway create --name MU_AG --resource-group MY_RG --sku Standard_v2 --subnet $AGSubnetId --location westeurope --public-ip-address <Public IP name> --private-ip-address <Private Subnet IP ID>
```

My az CLI version is 2.9.1

**Comment by alexpilon666** [comment]
@yonzhan , why the "Feature Request" tag ? This is not a feature request, but a bug, as the `--private-ip-address` argument already exists, but doesn't do what it is supposed to do.

**Comment by haroldrandom** [comment]
From the service side, we cannot specify both private IP and public IP while creating an application gateway. 
If we did so, it would return `ApplicationGatewayFrontendIpCannotHavePublicIpAndSubnet` error.

In current CLI implementation, it will ignore the value of `--private-ip-address` once `--public-ip-address` is specified without warning:
https://github.com/Azure/azure-cli/blob/4dba577c2dcede9fd81ed35640ef0ed0b6f0391a/src/azure-cli/azure/cli/command_modules/network/_template_builder.py#L15-L28

I looked into the usgae statistic, the usage us slow, I think we could correct this by breaking it.
